console/pom.xml

```diff
@@ -93,13 +93,6 @@
                     <exclude>static/console-fe/.vscode/**</exclude>
                 </excludes>
             </resource>
-            <resource>
```

Comment:
Why remove it

Reply:
The VersionUtils class is in the common module, but the nacos-version.txt file is in the console module, and the version number obtained in the client module using VersionUtils.VERSION is null, so now I need to move the nacos-version.txt file along with the relevant code to the common module.

Reply:
ok,